Assets/_Application/Samplers/OpticalRaycaster.cs

```diff
@@ -61,6 +61,24 @@ public void GetWorldPointsAsync(Vector3[] screenPoints, Action<Vector3[], bool>
             }
         }
 
+        public void GetWorldPointFromDirectionAsync(Vector3 worldPosition, Vector3 direction, Action<Vector3, bool> callback, int cullingMask = defaultRaycastLayers)
+        {
+            if (activeRequests.Count > maxRequests)
```

Comment:
Lees ik dit goed dat de callback alleen wordt aangeroepen als deze methode meermaals wordt uitgevoerd? Ik zou bij een callback verwachten dat die altijd uitvoert als de methode klaar is - of begrijp ik dit verkeerd?

Reply:
Check!